google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -460,9 +461,24 @@ def deploy_to_local_endpoint(
 
         Args:
             artifact_uri (str):
-                Optional. The Cloud Storage path to the directory containing the Model artifact
-                and any of its supporting files. The AIP_STORAGE_URI environment variable will
-                be set to this uri if given; otherwise, an empty string.
+                Optional. The path to the directory containing the Model artifact and any of its
```

Comment:
General comment - 

Do we need the artifact_workdir param? Does the user need to specify this? (Can we just choose a default for the user?)

If artifact_uri is a GCS path - then it'll be passed directly to Predictor.load().
If artifact_uri is a local path - then we'll mount that path to a default path (chosen by us), and the mount path gets passed to Predictor.load().

Does that work?

Reply:
I got rid of the artifact_workdir parameter. In download_model_artifacts, I copied the files to the current workdir too if it's a local path. Therefore users don't need to have two code paths in their `Predictor.load` (they can assume the files are in the current workdir). Let me know if you have any comments. Thanks!

Reply:
Cool, that works.